charmhelpers/contrib/openstack/templates/openstack_https_frontend

```diff
@@ -6,8 +6,18 @@ Listen {{ ext_port }}
 <VirtualHost {{ address }}:{{ ext }}>
     ServerName {{ endpoint }}
     SSLEngine on
-    SSLProtocol +TLSv1 +TLSv1.1 +TLSv1.2
-    SSLCipherSuite HIGH:!RC4:!MD5:!aNULL:!eNULL:!EXP:!LOW:!MEDIUM
+
+    # This section is based on Mozilla's recommendation
+    # as the "intermediate" profile as of July 7th, 2020.
+    # https://wiki.mozilla.org/Security/Server_Side_TLS
+    SSLProtocol all -SSLv3 -TLSv1 -TLSv1.1
+    SSLCipherSuite ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384
+    SSLHonorCipherOrder off
+    # SSLSessionTickets is supported by Apache 2.4.11 or later.
+    # The following line can be enabled when trusty support is dropped.
+    # https://httpd.apache.org/docs/trunk/mod/mod_ssl.html#sslsessiontickets
+    # SSLSessionTickets off
```

Comment:
These lines are commented out already, please remove them from the template

Reply:
From the history quoted in the description, it looks like SSLSessionTickets were removed a while ago as a direct option, is there any reason why we'd re-add this config option?